grades/tests.py

```diff
@@ -1,3 +1,50 @@
-# from django.test import TestCase
+import pytest
+from grades.models import Grade, Course
+from django.contrib.auth.models import User
 
-# Create your tests here.
+
+@pytest.mark.django_db
+class TestGrade:
+    def test_addGradeUserCourse(cls):
+        out = Grade.objects.filter(semester='A')
+        assert list(out.values_list('user', 'course', 'grade', 'semester', 'year')) == [
+            (1, 1, 85, 'A', 2021),
+        ]
+
+    def test_removeGrade(cls):
+        user3 = User.objects.create_user(username="idan", password="123456")
+        c = Course(course_name="NLP", credinitials="3")
+        c.save()
+        g = Grade(user=user3, course=c, grade=90, semester='A', year=2021)
+        g.save()
+        g = Grade.objects.get(id=2)
+        g.delete()
+        out = Grade.objects.all()
+        assert list(out.values_list('user', 'course', 'grade', 'semester', 'year')) == [
+            (1, 1, 85, 'A', 2021),
+            (1, 1, 20, 'C', 2020),
+            (2, 3, 90, 'A', 2021),
+        ]
+
+
+@pytest.mark.django_db
+class TestCourse:
+    def test_addCourse(cls):
+        c = Course(course_name="java", credinitials="3")
+        c.save()
```

Comment:
same comment as above

Reply:
@simzacks Use it now it , I update now the commit